compiler/rustc_typeck/src/check/fn_ctxt/checks.rs

```diff
@@ -226,6 +227,20 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                     String::from("()"),
                     Applicability::MachineApplicable,
                 );
+            } else if sugg_tuple_wrap {
+                let (first, last) = match args {
+                    [first, .., last] => (first, last),
+                    _ => bug!("shouldn't be called without 2-or-more-tuple"),
+                };
+
+                err.multipart_suggestion(
+                    "expected a tuple composed of these values; create it with parentheses",
```

Comment:
Something like the following would be a more idiomatic message:

```suggestion
                    "use parentheses to construct a tuple",
```